packages/manager/src/mocks/serverHandlers.ts

```diff
@@ -548,6 +548,122 @@ const parentAccountNonAdminUser = accountUserFactory.build({
   username: 'NonAdminUser',
 });
 
+const cloudpulse = [
+  http.get('*/monitor/services', () => {
```

Comment:
any mock response for error?

Reply:
Usually, in test files those cases are covered by sending response that might lead to errors . So we are going to test the error handling in the components